dlk/tests/test_code_generation.py

```diff
@@ -366,9 +383,11 @@ def codegen_cpu(self,
         if not use_run_test_script:
             if cpu_name == 'x86_64':
                 percent_failed = self.run_library(generated_lib, input_path, expected_output_path)
-            else:
+            elif cpu_name == 'arm' or cpu_name == 'arm_fpga':
                 percent_failed = \
                     self.run_library_on_remote(FPGA_HOST, output_path, generated_lib, input_path, expected_output_path)
+            else:
+                return
```

Comment:
what do we expect here?
maybe we should do:
```
elif cpu_name == 'aarch64':
    # comment explaining why
    return
eles:
    self.fail("Unexpected cpu_name: %s" % cpu_name)
```

Reply:
Thanks, I updated! 👍